administrator/components/com_templates/Controller/OverridesController.php

```diff
@@ -0,0 +1,56 @@
+<?php
+/**
+ * @package     Joomla.Administrator
+ * @subpackage  com_templates
+ *
+ * @copyright   Copyright (C) 2005 - 2018 Open Source Matters, Inc. All rights reserved.
+ * @license     GNU General Public License version 2 or later; see LICENSE.txt
+ */
+namespace Joomla\Component\Templates\Administrator\Controller;
+
+defined('_JEXEC') or die;
+
+use Joomla\CMS\MVC\Controller\BaseController;
+
+/**
+ * Template overrides controller class.
+ *
+ * @since  __DEPLOY_VERSION__
+ */
+class OverridesController extends BaseController
+{
+  /**
+	 * Fetch and report updates in \JSON format, for A\JAX requests
+	 *
+	 * @return void
+	 *
+	 * @since 2.5
+	 */
+	public function ajax()
+	{
+		$app = $this->app;
+
+		if (!\JSession::checkToken('get'))
+		{
+			$app->setHeader('status', 403, true);
+			$app->sendHeaders();
+			echo \JText::_('JINVALID_TOKEN');
+			$app->close();
+		}
+
+		// Created only for test.
+		$session = \JFactory::getSession();
+
+		$result = array();
+		$updates = $session->get('override.result');
```

Comment:
So when the user makes the update and logs out, then the information is gone as we store it in the session?